kubernetes-job/src/chart/templates/_helpers.tpl

```diff
@@ -3,7 +3,15 @@ Since this is a template chart, the chart name will be filled in by the user and
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "application.fullname" -}}
-{{- .Chart.Name | trunc 63 | trimSuffix "-" }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- define "timestamp" -}}
+{{ now | date "20060102150405" }}
+{{- end }}
+
+{{- define "application.timestamped-name" -}}
```

Comment:
We create a timestamped name for the job so that we won't have collisions (though it appears kubernetes may append a random suffix anyway so this might be optional).